pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -136,8 +136,8 @@ func (m *MultiNodeConsolidation) firstNConsolidationOption(ctx context.Context,
 		// required
 		replacementHasValidInstanceTypes := false
 		if cmd.Action() == ReplaceAction {
-			cmd.replacements[0].InstanceTypeOptions = filterOutSameType(cmd.replacements[0], candidatesToConsolidate)
-			replacementHasValidInstanceTypes = len(cmd.replacements[0].InstanceTypeOptions) > 0
+			cmd.replacements[0].InstanceTypeOptions, err = filterOutSameType(cmd.replacements[0], candidatesToConsolidate)
```

Comment:
Do we ever expect this error to be non-nil since min values is still already validated in compute consolidation? It doesn't hurt to have this check, but if there is a new non-nil scenario, I'm wondering if we should still silently fail here.

Reply:
It could definitely be non-nil. If we filter out the same type and this breaks the minValues requirement, then we should fail

Reply:
Got it, makes sense to me 👍 When I was reading the original implementation I missed that `filterByPriceWithMinValues` would return an empty slice when violated so I thought this was a new error check.